test/runtests.jl

```diff
@@ -4,7 +4,8 @@
 using RoME
 using Test
 using TensorCast
-using Manifolds
+import Manifolds
+using Manifolds: ProductManifold, SpecialEuclidean, ProductRepr, SpecialOrthogonal, TranslationGroup
```

Comment:
Just note I usually keep test files "independent" so that we can run any one alone without needing runtests.jl.  This is much fast when debugging a specific issue.